src/cmd/build.js

```diff
@@ -13,8 +14,10 @@ const log = createLogger(__filename);
 
 
 export default function build(
-    {sourceDir, artifactsDir}: Object,
-    {manifestData, fileFilter}: Object = {}): Promise {
+    {sourceDir, artifactsDir, asNeeded: rebuildWhenSourceChanges}: Object,
```

Comment:
I find the renaming of `asNeeded` to `rebuildWhenSourceChanges` in the above function parameters a bit hard to read (and the flow annotations makes it even a bit harder then usual), I propose to use it directly as `asNeeded` (without renaming it), or to rename it to `rebuildWhenSourceChanges` into the function body.